elephant/test/test_statistics.py

```diff
@@ -701,7 +702,8 @@ def test_trim_as_convolve_mode(self):
             for trim in (False, True):
                 rate_centered = statistics.instantaneous_rate(
                     spiketrain, sampling_period=sampling_period,
-                    kernel=kernel, cutoff=cutoff, trim=trim)
+                    kernel=kernel, cutoff=cutoff, trim=trim,
+                    center_kernel=True)
```

Comment:
for better readability of the unittest, furthermore  the default `center_kernel=True` might be changed in the future